canonical-web-and-design/maas-ui
# Add yarn target build-all to build all projects in the root.

## Done
Add new yarn target `build-all`, which cleans, builds, and copies production bundles to `/build`.

## QA
* Run `yarn build-all` from the project root.
* Wait a while  👀 
* Production bundles should appear in `/build/ui` and `/build/legacy`.
